app/notify_client/service_api_client.py

```diff
@@ -6,6 +6,13 @@
 
 class ServiceAPIClient(NotifyAdminAPIClient):
 
+    def _delete_template_cache_for_service(self, service_id):
+        templates_for_service = self.get_service_templates(service_id)['data']
+        redis_client.delete(*map(
+            'template-{}-version-None'.format,
+            [x['id'] for x in templates_for_service],
+        ))
```

Comment:
I’m sure you told me something about Guido not liking `map` at one point
```suggestion
        redis_client.delete(*[
            f'template-{template["id"]}-version-None'
            for template in templates_for_service
        ])
```